modules/sdk/src/main/scala/org/tessellation/sdk/http/p2p/clients/L0GlobalSnapshotClient.scala

```diff
@@ -56,5 +57,11 @@ object L0GlobalSnapshotClient {
 
         PeerResponse[F, Signed[GlobalIncrementalSnapshot]](s"global-snapshots/$hash")(client)
       }
+
+      def getHash(ordinal: SnapshotOrdinal): PeerResponse[F, Hash] = {
+        import org.http4s.circe.CirceEntityCodec.circeEntityDecoder
+
+        PeerResponse[F, Hash](s"global-snapshots/hash/${ordinal.value.value}")(client)
+      }
```

Comment:
if `NotFound` is an expected http status (I believe it is as someone can request for future ordinal which does not exist yet), then the client should handle that status accordingly. IMO the returned type should be `PeerResponse[F, Option[Hash]]` and `NotFound` should be mapped to `None` then

Reply:
The route is returning an `Option[Hash]` now.